CHANGELOG.md

```diff
@@ -1,8 +1,14 @@
 # Change Log
 
-## [v1.2.1](https://github.com/grpc-ecosystem/grpc-gateway/tree/v1.2.1) (2017-04-16)
+## [v1.2.2](https://github.com/grpc-ecosystem/grpc-gateway/tree/v1.2.2) (2017-04-17)
 **Merged pull requests:**
 
+- metadata: fix properly and change to Outgoing [\#361](https://github.com/grpc-ecosystem/grpc-gateway/pull/361)
+
+## [v1.2.1](https://github.com/grpc-ecosystem/grpc-gateway/tree/v1.2.1) (2017-04-17)
```

Comment:
I'm just taking verbatim https://github.com/skywinder/github-changelog-generator output here. It's not clear why it detected the different date.